src/System.Memory/ref/System.Memory.cs

```diff
@@ -53,6 +53,9 @@ public static partial class MemoryExtensions
         public static int IndexOf<T>(this System.Span<T> span, System.ReadOnlySpan<T> value) where T : System.IEquatable<T> { throw null; }
         public static int IndexOf<T>(this System.Span<T> span, T value) where T : System.IEquatable<T> { throw null; }
         public static bool IsWhiteSpace(this System.ReadOnlySpan<char> span) { throw null; }
+#if FEATURE_PLATFORM_SPAN
```

Comment:
I think we should do the change to pick up the netstandard2.0 version via the package harvesting instead. We are going to be adding a bunch of new APIs to this contract, and it would be pretty ugly to have all new APIs added under ifdefs.

Reply:
Unfortunately we can't. If we harvest netstandard2.0 and netstandard1.1 then we don't have a netstandard configuration which is required for System.IO.Pipelines and System.Net.WebSockets.WebSocketProtocol to build. Switching those ProjectReferences to PackageReferences doesn't work either as we currently don't support restoring packages. In fact it works for S.N.W.WebSocketProtocol because it doesn't use any types from System.Memory in the contract.

I guess we either need to enable the package restore mechanism or move S.IO.Pipelines to netcoreapp and harvest the netstandard configuration.

cc @ericstj @weshaggard

Reply:
Ugh. If we really have to be building this - I would just use `netstandard` define for this instead of `FEATURE_PLATFORM_SPAN`. `netstandard` is more self-explanatory and it is what we are using in other similar places.

Reply:
> Unfortunately we can't. 

Yes you can.  If you want to stop building a ref and harvest it for a package you just need to harvest it for the build as well.  To do that you would make sure the the appropriate external project restores it and gets it in the ref path, eg: https://github.com/dotnet/corefx/blob/master/external/netstandard/netstandard.depproj.

Note that due to build-from-source requirements you must keep the netcoreapp configuration building from source.

Reply:
Awesome, that's exactly what I was looking for. Thanks so much!

Reply:
As an aside it is also probably safer to stop building the netstandard ref for System.Memory as we don't want to accidentally get into trouble by shipping new APIs for it.

Reply:
@weshaggard we could say the same for any package which has a netstandard ref and is inbox on any framework.  Do you think we should do so for multiple projects?

Reply:
That may be a good thing to do.

Reply:
sure. created https://github.com/dotnet/corefx/issues/30960 to track it.